charts/datasaur/values.yaml

```diff
@@ -160,16 +162,159 @@ jobs:
+                                - B
+                          reducer:
+                            params: []
+                            type: last
+                          type: query
+                    datasource:
+                        type: __expr__
+                        uid: __expr__
+                    expression: A
+                    hide: false
+                    intervalMs: 1000
+                    maxDataPoints: 43200
+                    reducer: mean
+                    refId: B
+                    settings:
+                        mode: dropNN
+                    type: reduce
+                - refId: C
+                  relativeTimeRange:
+                    from: 600
+                    to: 0
+                  datasourceUid: __expr__
+                  model:
+                    conditions:
+                        - evaluator:
+                            params:
+                                - 1
+                            type: gt
+                          operator:
+                            type: and
+                          query:
+                            params:
+                                - C
+                          reducer:
+                            params: []
+                            type: last
+                          type: query
+                    datasource:
+                        type: __expr__
+                        uid: __expr__
+                    expression: B
+                    hide: false
+                    intervalMs: 1000
+                    maxDataPoints: 43200
+                    refId: C
+                    type: threshold
+              noDataState: OK
+              execErrState: Error
+              for: 1m
+              annotations:
+                description: |-
+                    Node: {{ $labels.node_name }}
+                    Pod: {{ $labels.pod }}
+                summary: |-
+                    {{ $labels.alertname }}
+                    {{ $labels.app }} has more than 5 error request in the last 10 minute
+              labels:
+                client: datasaur
+              isPaused: false
+    notification.yaml: |
+      apiVersion: 1
+
+      policies:
+        - orgId: 1
+          receiver: cp_1
+          matchers: 
+            - client = datasaur
+          object_matchers:
+            - ['client', '=', 'datasaur']
+
   auth:
     username: null
     password: null
   grafana:
     enabled: false
     grafana.ini:
       server:
-        domain: "{{ .Values.ingress.host }}"
-        root_url: "%(protocol)s://%(domain)s:%(http_port)s/grafana/"
+        root_url: "%(protocol)s://%(domain)s/grafana/"
         serve_from_sub_path: true
+    extraConfigmapMounts:
+      - name: grafana-alert-provisioning
+        mountPath: /etc/grafana/provisioning/alerting
+        configMap: grafana-alert-provisioning
+        subPath: ""
+        readOnly: true
+    image:
+      tag: 9.5.2
```

Comment:
why do we put this 9.5.2 tag here?

Reply:
because file provisioning baru disupport di versi 9 ke atas, defaultnya adalah pake versi 8 di loki-stack chart itu, that's why i bumped up the version @tonoman3g